lib/vector-config-common/src/schema/gen.rs

```diff
@@ -42,6 +42,7 @@ impl SchemaSettings {
 
     /// Creates a `Visitor` from the given closure and appends it to the list of
     /// [visitors](SchemaSettings::visitors) for these `SchemaSettings`.
+    #[allow(rustdoc::private_intra_doc_links)]
```

Comment:
Are we not able to "fix" these?

Reply:
I wasn't able to. The recommendation to pass `--document-private-items` did not fix the error, so I figured we could defer for now.